connector_search_engine/models/se_binding.py

```diff
@@ -72,7 +72,7 @@ def create(self, vals):
 
     def write(self, vals):
         not_new = self.browse()
-        if "active" in vals and not vals["active"]:
```

Comment:
I'm not sure this is enough. IMO in case a record is re-activated the json data should be recompute first -> this will lead to proper state.

Reply:
You are right.
What we need to do is if the field is we have vals["active"] is to call jobify_recompute_json(force_export=True) and this job will update the json and then set it to_update